### PR TITLE
WIP: Politique de confidentialité: plus de parties prenantes et justifications

### PR DESCRIPTION
**Pull request by r0one**
Before merging, we will have to implement the right for the user to decide wether he wants to display his name on bookings (issue #7 ).

### CODE REVIEW

File: front/pages/policy.vue [hunk 1]
```diff
@@ -51,40 +75,78 @@
...
+      association utilisatrice le cas échéant). Il est nécessaire de traiter ces
+      informations pour fournir le service d'emploi du temps des salles.
+    </p>
+    <p>
+      Si vous ne souhaitez pas utiliser le portail des assos pour transmettre
+      ces données au PAE-UTC, vous pouvez le faire en contactant le PAE-UTC à
+      son adresse mail :
+      <a href="mailto:poleae@assos.utc.fr">poleae@assos.utc.fr</a>. Ces données
+      seront entrées par le PAE-UTC sur ce site et seront soumises à la présente
+      polique de protection des données en ce qui concerne leur traitement.
```
Comment (lvaroqui) [inline front/pages/policy.vue:94]: This is not implemented and is not planed to be.

Comment (r0one) [reply front/pages/policy.vue:94]: It is illegal to force users to use the portal just for booking a room. We can force associations, but not persons. If we want to force people that don't belong to associations (people in MU0x), we have to sign a deal with UTC.

Comment (r0one) [reply front/pages/policy.vue:94]: Each mandatory treatment must be justified and obviously the PAE has got the technical capacity to fill in a schedule.

**Review by lvaroqui — APPROVED** [review]
I think this is good to merge, @CdRom1 is ok for you ?

**Comment by r0one** [comment]
I think we made a good compromise ^^

**Pull request merged**